lesson-5/Alnahdi - Week 5/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample Project</title>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:400,700" rel="stylesheet">
+  </head>
+
+  <body>
+
+
+    <!-- PUT THE SAMPLE IMAGE AND YOUR CODE HERE -->
+    <div>
+    	<img id="img" src="sample-project.jpg"/>
+    	<div id="contents">
+    		<h3>My Awesome Project</h3>
+    		<p>This is the description of my project. It's a
+    		really awesome project.</p>
+    		<div id="button">
```

Comment:
This is a great work around for getting the link to display like a block element. For less markup (which is generally better practice), consider giving the `<a></a>` itself the id of `button` and using CSS to make it a block element.